doc/releases/changelog-dev.md

```diff
@@ -4,7 +4,53 @@
 
 <h3>New features since last release</h3>
 
-* `DefaultQubit` devices now natively support parameter broadcasting.
+* The gradient transform `qml.gradients.param_shift` now accepts the new Boolean keyword
+  argument `broadcast`. If it is set to `True`, broadcasting is used to compute the derivative.
+  [(#2749)](https://github.com/PennyLaneAI/pennylane/pull/2749)
+
+  For example, for the circuit
+
+  ```python
+  x, y = np.array([0.4, 0.23], requires_grad=True)
+
+  with qml.tape.QuantumTape() as tape:
```

Comment:
Oh, what I originally meant here was that a QNode could be passed to `qml.gradients.param_shift` as the first argument (at least based on the docstring of the function, haven't tested it myself).

Reply:
If that's the case, it would likely be preferred. I'm still unsure if we usually would like to showcase tapes in user-facing examples, as mostly they are advanced user/developer facing.

Reply:
I see! I agree that the QNode functionality (and yes, it works) should be preferred. The problem for this case is, that the changes of this PR are not visible at all beyond the user input `broadcast=True` when looking at the QNode level. 
While that is nice UX-wise, I felt like the tape-level effects might be more interesting? 
On the QNode-level, the changelog entry would look like this:

The `qml.gradients.param_shift` transform now takes a keyword argument `broadcast` that enables
the use of parameter broadcasting under the hood to evaluate the gradient:
```py
some QNode
```
```pycon
>>>qml.gradients.param_shift(circuit, broadcast=True)(*args)
#the gradient
```

Reply:
Anything needing lower-level functionality can exist, but does not need to be directly exposed to the release notes, as this pretty much guarantees some degree of support for the given use.

It may be worth exploring this functionality later with @albi3ro upcoming work on the intermediate circuit class, as that is something that is more forward facing.

Reply:
Alright, I'll reduce the changelog accordingly :)